src/processing/script/index.js

```diff
@@ -152,9 +152,10 @@ export function processScript (src, withHeader) {
     withHeader = withHeader && !isObject && !isArrayDataScript(ast);
 
     var changes   = transform(ast);
+    var isStrict  = hasStrictDirective(ast);
```

Comment:
All we need to do is to bump `"use strict";` to the top of the file, right?
If so, we can do it in `postprocess`, it has nothing to do with header